app/src/main/java/com/example/quizacademytask/StackListFragment.kt

```diff
@@ -67,7 +70,13 @@ class StackListFragment : Fragment(), SimpleAdapter.OnItemClickListener {
         courseJSON = ""
         swipeContainer = binding.swipeContainer
         idlingResource = CountingIdlingResource("API")
+
+        //Set the toolbar
         binding.toolbarStackList.title = "Topics"
+        (activity as MainActivity).setSupportActionBar(toolbar)
+        toolbar.setOnClickListener(View.OnClickListener {
```

Comment:
1. this looks not very useful, 2. you could shorten such click listeners by using lambdas 
```kotlin
...setOnClickListener {
}
```